lib/roby/interface/rest/server.rb

```diff
@@ -18,6 +18,11 @@ class Server
                 # @return [String]
                 attr_reader :host
 
+                # The host the server routes through
+                #
+                # @return [String]
+                attr_reader :main_route
```

Comment:
The documentation is very confusing. I'd prefer `The path under which this API will be available, e.g. /api`

Reply:
done